Cargo.toml

```diff
@@ -50,7 +51,7 @@ members = [
     "test-rt/test-unit-core",
     "test-rt/test-onnx-core",
     "test-rt/test-nnef-cycle",
-    "test-rt/test-tflite", 
+    "test-rt/test-tflite", "examples/face_similarity_arcface_onnx", 
 ]
```

Comment:
Can you adjust the formatting / ordering please ?

Reply:
yes, i will update it shortly